keyboards/ai03/andromeda/readme.md

```diff
@@ -0,0 +1,10 @@
+# ai03 Andromeda
+
+* Keyboard Maintainer: [ai03](https://github.com/ai03-2725)
+* Hardware Supported: Andromeda PCB
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ai03/andromeda:default
+
```

Comment:
Could you please add instructions here on how to reset the keyboard into bootloader mode?